tests/utils/shopper.js

```diff
@@ -56,4 +60,38 @@ export const shopper = {
 
 		await expect( page ).toMatchElement( 'h1', { text: title } );
 	},
+
+	/**
+	 * Override the @woocommerce/e2e-utils `emptyCart` method to fix the
+	 * ReferenceError issue and remove the cart items.
+	 *
+	 * @todo Remove shopper.emptyCart overload once the upstream  is fixed
```

Comment:
is this something we should contribute to WooCommerce or just local to us?

Reply:
Yes. I submitted a PR and it was merged last week: https://github.com/woocommerce/woocommerce/pull/31977.

Reply:
Nice! Should we add a @todo to remove it when next WooCommerce e2e utils get released?

Reply:
I added the todo already.

Reply:
🤦 me being me... sorry for the oversight! :)